battleship.rb

```diff
@@ -0,0 +1,84 @@
+# require 'pry'
+# attr_accessor :radar, :ocean
+#Generates the board, called ocean
+x_axis = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
+y_axis = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']
+@ocean = x_axis.product(y_axis).each do |combo|
```

Comment:
No need to define instance variables here since they're not particular to an instance.